index.html

```diff
@@ -16,15 +16,15 @@
       <div class="post">
         <a href="{{ post.url | prepend: site.baseurl }}" class="post-link">
           <p class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</p>
-          <h3 class="h2 post-title">{{ post.title }}</h3>
+          <h3 class="h2 post-title">{{ post.title | xml_escape }}</h3>
         </a>
-        <p class="post-summary">
+        <div class="post-summary">
```

Comment:
Hopefully this doesn't break the look, but htmlproofer complains about the closing /p.